packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -20,13 +22,20 @@ import {SpectrumFieldProps} from '@react-types/label';
 function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
   let {
     label,
-    labelPosition = 'top',
+    labelPosition = 'top' as LabelPosition,
     labelAlign,
     isRequired,
     necessityIndicator,
     includeNecessityIndicatorInAccessibilityName,
+    validationState,
+    description,
+    errorMessage,
+    isDisabled,
+    showIcon,
     children,
     labelProps,
+    descriptionProps,
```

Comment:
it seems odd to me that Field doesn't `useField`, that `Textfield` uses `useTextfield` which uses `useField`. 
apologies if this was discussed and I forgot

Reply:
I did feel a bit funny while implementing it too but it makes more sense after reading this from the issue notes:
> Once we have this hook, it should replace the call to useLabel in the hooks for all components where help text is supported. These are: useTextField, useSelect, useComboBox, useRadioGroup, and useCheckboxGroup. descriptionProps and errorMessageProps will need to be added to the return types for all of these hooks, and passed through to the Field component in Spectrum.